lib/src/common/variables.js

```diff
@@ -214,21 +213,49 @@ export const componentTokens = {
   },
   header: {
     backgroundColor: globalTokens.white,
-    underlinedColor: globalTokens.black,
-    fontColor: globalTokens.black,
-    backgroundColorMenu: globalTokens.white,
-    fontColorMenu: globalTokens.black,
-    hamburguerColor: globalTokens.black,
-    hoverHamburguerColor: globalTokens.mediumGreyBlack,
-    overlayColor: globalTokens.softBlack,
+    customContentFontFamily: "",
+    customContentFontStyle: globalTokens.type_normal,
+    customContentFontColor: "",
+    customContentFontSize: "",
+    customContentFontWeight: globalTokens.type_regular,
+    dropdownBackgroundColor: globalTokens.transparent,
+    dropdownHoverBackgroundColor: globalTokens.transparent,
     focusColor: globalTokens.blue,
```

Comment:
Where is the `focusColor` token applied?

Reply:
`focusColor` is applied to the hamburguer button when is clicked or focused.

Reply:
Should we call it then `hamburguerFocusColor` instead? :hamburger:

Reply:
`focusColor` renamed to `hamburguerFocusColor` in last commit.